README.md

```diff
@@ -35,19 +38,42 @@ services:
        - "./storage/mysql:/var/lib/mysql"
     environment:
       MYSQL_RANDOM_ROOT_PASSWORD: "yes"
-      MYSQL_DATABASE: "glpi"
-      MYSQL_USER: "glpi"
-      MYSQL_PASSWORD: "glpi"
+      MYSQL_DATABASE: ${GLPI_DB_NAME}
+      MYSQL_USER: ${GLPI_DB_USER}
+      MYSQL_PASSWORD: ${GLPI_DB_PASSWORD}
```

Comment:
Did you make sure that this work with the example `glpi.env` file?

Interpolation only works from a `.env` file, values from `glpi.env` should not be available here unless I am missing something.

<img width="1332" height="456" alt="Image" src="https://github.com/user-attachments/assets/185cef54-1347-401c-8b9c-670227b73a0f" />